### ISSUE TITLE
Slate crashes when invoking a chain

### ISSUE THREAD

**Issue by lopopolo** [jigish/slate#72]
Slate dies when I try to trigger this binding:

```
bind pad5:ctrl,shift  chain corner bottom-left resize:2*screenSizeX/5;screenSizeY/2 | corner bottom-right resize:2*screenSizeX/5;screenSizeY/2 | corner top-left resize:2*screenSizeX/5;screenSizeY/2 | corner top-right resize:2*screenSizeX/5;screenSizeY/2
```

This binding is a composite of these bindings which work correctly

```
bind pad1:ctrl,shift  corner bottom-left  resize:2*screenSizeX/5;screenSizeY/2
bind pad3:ctrl,shift  corner bottom-right resize:2*screenSizeX/5;screenSizeY/2
bind pad7:ctrl,shift  corner top-left     resize:2*screenSizeX/5;screenSizeY/2
bind pad9:ctrl,shift  corner top-right    resize:2*screenSizeX/5;screenSizeY/2
```

**Comment by jigish** [comment]
will be fixed in the next version (0.5.10). this was a bug introduced by my conversion to ARC.

**Comment by jigish** [comment]
new version is out.
